__tests__/app.test.js

```diff
@@ -23,6 +23,161 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("200: Responds with all the data from the articles tables", () => {
+    const expectedResult = [
```

Comment:
This makes your test file huuuuge and unruly. Maybe just check that the length of the array we get back is what we expect it to be and then do a forEach, checking that all of them have the key and data type we'd expect to see with a toMatchObject :)

Reply:
I'd like to also see a separate test for checking that the sort DESC is working with the above method, tho. Jest Sorted easily sorts you out for that, though: https://www.npmjs.com/package/jest-sorted